### PR TITLE
Supporting multivariate flags

### PR DESCRIPTION
**Pull request by lukewalczak**
Solves #12.

### CODE REVIEW

**Comment by toddw** [comment]
Can you please add an example of how we would use this to README.md?

File: README.md [hunk 1]
```diff
@@ -79,3 +80,6 @@ export default class Home extends Component {
 }
 
 ```
+### Support for multivariate flags
+React-Launch-Darkly supports five types of flags which are:
+booleans, strings, numbers, JSON objects, and JSON arrays. `renderFeatureCallback` prop function takes an argument which is value of flag. That's the way you can use multivariate feature flags.
```
Comment (jacobmoretti) [inline README.md]: Can we say this instead, "The prop function, `renderFeatureCallback`, passes the value of the flag as an argument. Then when you define your render feature callback, you can decide if you need it or not. This provides the flexibility for supporting both simple boolean feature flags and multivariate flags where you need to do something more complex with the flag's value."

Comment (lukewalczak) [inline README.md]: Sure, I like it !

File: src/components/FeatureFlagRenderer.js [hunk 2]
```diff
@@ -59,18 +59,18 @@ export default class FeatureFlagRenderer extends Component {
     const ldClient = ldBrowserInit(clientId, user);
 
     ldClient.on("ready", () => {
-      const showFeature = ldClient.variation(flagKey, false);
+      const flagValue = ldClient.variation( flagKey, false);
+      const typeShowFeature = typeof flagValue;
```
Comment (jacobmoretti) [inline src/components/FeatureFlagRenderer.js]: For readability and consistency, can you rename this variable to: `typeFlagValue` to match the rename of `showFeature` -> `flagValue`.

File: src/lib/launchDarkly.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ export function ldBrowserInit (key, user) {
   return launchDarklyBrowser.initialize(key, user);
 }
 
-export function ldOverrideFlag(flagKey) {
+export function ldOverrideFlag(flagKey, typeShowFeature) {
```
Comment (jacobmoretti) [inline src/lib/launchDarkly.js]: Lets update the argument here too to match: `typeFlagValue`.

File: src/components/FeatureFlagRenderer.js (hunk 2 above)
Comment (lukewalczak) [inline src/components/FeatureFlagRenderer.js]: Forgot to change it. Thanks !

File: src/lib/launchDarkly.js (hunk 3 above)
Comment (lukewalczak) [inline src/lib/launchDarkly.js]: Done

**Comment by jacobmoretti** [comment]
@lukewalczak Awesome! Thanks for the updates

**Pull request merged**